_includes/pbc/all/install-features/202407.0/marketplace/install-the-marketplace-merchant-feature.md

```diff
@@ -218,6 +218,55 @@ class MessageBrokerConfig extends SprykerMessageBrokerConfig
 
 ```
 
+5. Enable the order expenses for the merchant payout:
+
+**src/Pyz/Zed/SalesPaymentMerchant/SalesPaymentMerchantConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\SalesPaymentMerchant;
+
+use Spryker\Shared\Shipment\ShipmentConfig;
+use Spryker\Zed\SalesPaymentMerchant\SalesPaymentMerchantConfig as SprykerSalesPaymentMerchantConfig;
+
+class SalesPaymentMerchantConfig extends SprykerSalesPaymentMerchantConfig
+{
+    /**
+     * @var bool
+     */
+    protected const ORDER_EXPENSE_INCLUDED_IN_PAYMENT_PROCESS = true;
+    
+    /**
+     * @var array<string, list<string>>
+     */
+    protected const EXCLUDED_EXPENSE_TYPES_FOR_STORE = [
+        'YOUR_STORE_NAME' => [ShipmentConfig::SHIPMENT_EXPENSE_TYPE],
+    ];
+}
+```
```

Comment:
requires https://spryker.atlassian.net/browse/ACP-3558 to be released first, right?